pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -108,32 +106,31 @@ def test_get_pretranslations_fluent_simple(
     pretranslated_string = "hello-world = gt_translation\n"
 
     response = get_pretranslation(fluent_entity, google_translate_locale)
-    assert response == (pretranslated_string, gt_user)
+    assert response == (pretranslated_string, "gt")
 
 
 @patch("pontoon.pretranslation.pretranslate.get_google_translate_data")
 @pytest.mark.django_db
 def test_get_pretranslations_fluent_empty(
-    gt_mock, fluent_resource, google_translate_locale, gt_user
+    gt_mock, fluent_resource, google_translate_locale
 ):
     # Entity.string is an empty Fluent string
     gt_mock.return_value = {
         "status": True,
         "translation": "gt_translation",
     }
 
-    fluent_string = 'hello-world = { "" }'
+    fluent_string = 'hello-world = { "" }\n'
     fluent_entity = EntityFactory(resource=fluent_resource, string=fluent_string)
-    pretranslated_string = "hello-world = gt_translation\n"
 
     response = get_pretranslation(fluent_entity, google_translate_locale)
-    assert response == (pretranslated_string, gt_user)
+    assert response == (fluent_string, "tm")
```

Comment:
The source `{ "" }` is now correctly parsed as not containing any literal text, and therefore isn't passed to Google Translate.